src/libtest/stats.rs

```diff
@@ -168,7 +168,7 @@ impl Stats for [f64] {
             let mut j = 0;
             // This inner loop applies `hi`/`lo` summation to each
             // partial so that the list of partial sums remains exact.
-            for i in 0..partials.len() {
+            for mut y in &mut partials {
                 let mut y: f64 = partials[i];
```

Comment:
Ouch – it's probably too late for me now; else I wouldn't have missed that one. Time to get some sleep.